src/libdredd/include_private/include/libdredd/mutate_visitor.h

```diff
@@ -103,11 +104,24 @@ class MutateVisitor : public clang::RecursiveASTVisitor<MutateVisitor> {
 
   // Yields the C++ constant-sized arrays, whose size expressions need to be
   // rewritten.
-  [[nodiscard]] const std::vector<const clang::VarDecl*>&
+  [[nodiscard]] const std::vector<const clang::DeclaratorDecl*>&
   GetConstantSizedArraysToRewrite() const {
     return constant_sized_arrays_to_rewrite_;
   }
 
+  // Yields the static assertions, whose expression need to be
+  // rewritten.
+  [[nodiscard]] const std::vector<const clang::StaticAssertDecl*>&
+  GetStaticAssertionsToRewrite() const {
+    return static_assertions_to_rewrite_;
+  }
+
+  // Yields the constant function arguments which need to be rewritten.
```

Comment:
Are these always for builtin functions? If so, could you change the name of the field and the getter to reflect that? E.g. `constant_builtin_function_arguments_to_rewrite` for the field, and `GetConstantBuiltinFunctionArgumentsToRewrite` for the getter. (I am OK with long names, as you have probably noticed!)

Reply:
changed